rules/windows/process_creation/win_susp_use_of_sqltoolsps_bin.yml

```diff
@@ -0,0 +1,32 @@
+title: SQL Client Tools PowerShell Session Detection
+id: a746c9b8-a2fb-4ee5-a428-92bee9e99060 
+status: experimental
+description: PowerShell execution through builtin SQL Server Management Studio "SQLToolsPS.exe" binary. Microsoft PS logging like
+  ScriptBlock logging function of PowerShell is not an option here, PS session caused by the binary won't be recorded/logged.
```

Comment:
Hello @uncleAntik! Thank you for your contribution (: 
Could you please change the description to make it one line?